test/extended/testdata/build-extended/bc-scripts-in-the-image.json

```diff
@@ -0,0 +1,34 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-image"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git",
+        "ref": "s2-assemble-only"
```

Comment:
Yes, because this branch contain only `assemble` script (I can't use `s2i` branch because it contains `assemble-runtime` and S2I will use it instead of image's one.)